draft-marchan-kdl2.md

```diff
@@ -265,10 +265,66 @@ KDL does not specify any restrictions on what implementations might do with
 these annotations. They are free to ignore them, or use them to make decisions
 about how to interpret a value.
 
-Additionally, the following type annotations MAY be recognized by KDL parsers
-and, if used, SHOULD interpret these types as follows:
+### Suffix Type Annotation
+
+When a ({{value}}) is a ({{number}}), it's possible to attach the type
+annotation as a "suffix", instead of prepending it between `(` and `)`. This
+makes it possible to, for example, write `10px`, `10.5%`, `512GiB`, etc., which
+are equivalent to `(px)10`, `(%)5`, and `(GiB)512`, respectively.
+
```

Comment:
Just for readability purposes, I think it's worth mentioning the `#` escape hatch here with a reference to the "Explicit Suffix Type Annotation" section, at least when it comes to types like `u32`. For example, maybe:

> To remove ambiguity, some suffixes must be prefixed with `#`: for example, `10.0u8` is invalid, but `10.0#u8` is. The full list of rules for invalid suffixes is clarified in the "Explicit Suffix Type Annotation" section.

Reply:
Ah, GitHub collapsed it as "outdated" because they're bad